ansible/group_vars/all.yml

```diff
@@ -31,8 +31,8 @@ stanchion_data_dir: "{{ encrypted_root }}/stanchion"
 # commcare-hq connects to an S3-compatible service, which is Riak CS
 s3_blob_db_enabled: "{{ 'true' if groups.get('riakcs') else '' }}"
 s3_blob_db_url: "http://{{ groups['proxy'][0] }}:{{ riakcs_port }}"
-s3_blob_db_access_key: "{{ hostvars[riakcs_control]['riak_key'] if s3_blob_db_enabled else '' }}"
-s3_blob_db_secret_key: "{{ hostvars[riakcs_control]['riak_secret'] if s3_blob_db_enabled else '' }}"
+s3_blob_db_access_key: "{{ hostvars[riakcs_control].get('riak_key', 'not-set') if s3_blob_db_enabled else '' }}"
+s3_blob_db_secret_key: "{{ hostvars[riakcs_control].get('riak_secret', 'not-set') if s3_blob_db_enabled else '' }}"
```

Comment:
I think I need more details to troubleshoot this further. The way it's designed the Riak CS machines should be configured and keys should be set by the time these `s3_blob_db_...` variables are used to write localsettings.py. I would need to know where in the process the failure occurred and what had happened before that to know why it failed.